packages/frontend/core/src/components/pure/workspace-slider-bar/workspace-card/index.tsx

```diff
@@ -86,63 +86,68 @@ const WorkspaceStatus = ({
 }) => {
   const isOnline = useSystemOnline();
 
-  // todo: finish display sync status
-  const [forceSyncStatus, startForceSync] = useDatasourceSync(
-    currentWorkspace.blockSuiteWorkspace
-  );
-
-  const setIsHovered = useSetAtom(hoverAtom);
+  const [syncEngineStatus, setSyncEngineStatus] = useState<
+    'syncing' | 'synced' | 'error'
+  >('syncing');
+
+  const syncEngine = useCurrentSyncEngine();
+
+  useEffect(() => {
+    const toStatus = (status?: SyncEngineStatus) => {
```

Comment:
btw, do we really need to convert `SyncEngineStatus` to strings here? seems the only difference is change `Retrying` to `error` but it will never shown to user. could we just keep it as it is?